moped-editor/src/utils/timelineTemplates.js

```diff
@@ -0,0 +1,155 @@
+export const returnSignalPHBMilestoneTemplate = projectId => {
```

Comment:
ok, so, there is currently only one template and it used by the four project types of concern (signal/phb mod/new).

to expand this to support multiple templates, we'll need to refactor here, and i assume that would also be the point at which it makes sense to start storing these in the DB?

Reply:
I went ahead and created this issue to capture this in the backlog: https://github.com/cityofaustin/atd-data-tech/issues/9395